code/raspi_sensor_tests/w1temperature.py

```diff
@@ -0,0 +1,30 @@
+import os
+from glob import glob
+from time import sleep
+
+os.system('modprobe w1-gpio')
+os.system('modprobe w1-therm')
+
+base_dir = '/sys/bus/w1/devices/'
+device_folder = glob(base_dir + '28*')[0]
+device_file = device_folder + '/w1_slave'
+
+def read_raw():
+	f = open(device_file)
+	lines = f.readlines()
+	f.close()
+	return lines
+
+def read_temp():
```

Comment:
there is a lib for it - [w1thermsensor](https://github.com/timofurrer/w1thermsensor)
parsing it manualy is kinda weird and error vulnerable